paper-tabs.html

```diff
@@ -230,7 +230,7 @@
       var tc = this.$.tabsContainer;
       var l = tc.scrollLeft;
       this.leftHidden = l === 0;
-      this.rightHidden = l === (tc.scrollWidth - tc.clientWidth);
+      this.rightHidden = l === Math.max(0, (tc.scrollWidth - tc.clientWidth));
```

Comment:
My reasoning for not doing so was that it gives more freedom to the expression that necessary to fix my issue: maybe it's possible to have `tc.scrollLeft > (tc.scrollWidth - tc.clientWidth) > 0` in a situation where the right scroll button shouldn't be hidden.

Perhaps that's being overly cautious though, I can update it to use `>=` if you think so.